.changelogs/6095.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Introduced locales and fixed bugs related to not handling properly some data",
+  "type": "fixed",
```

Comment:
Just as a formality - I know it will be v11, but adding a new option should increase `minor`. Maybe we can mention the same issue in the "added" and "fixed" sections.

Reply:
Changed within https://github.com/handsontable/handsontable/pull/8897/commits/58a44457a6c6f43a7cd5f9ddc61a2dc091780f20.